docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -36,7 +36,8 @@ You can add these Kubernetes annotations to specific Ingress objects to customiz
 |[nginx.ingress.kubernetes.io/backend-protocol](#backend-protocol)|string|HTTP,HTTPS,GRPC,GRPCS,AJP|
 |[nginx.ingress.kubernetes.io/canary](#canary)|"true" or "false"|
 |[nginx.ingress.kubernetes.io/canary-by-header](#canary)|string|
-|[nginx.ingress.kubernetes.io/canary-by-header-value](#canary)|string
+|[nginx.ingress.kubernetes.io/canary-by-header-value](#canary)|string|
+|[nginx.ingress.kubernetes.io/canary-match-strategy](#canary)|string|
```

Comment:
This annotation is specifically about, `canary-by-header-value`. What about calling it `canary-by-header-match-strategy`? (`canary-by-header-value-match-strategy` is a bit too verbose)

Reply:
done